client/signup/steps/import-from/wordpress/import-everything/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Title, SubTitle, NextButton } from '@automattic/onboarding';
+import { sprintf } from '@wordpress/i18n';
+import { Icon, check } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import React, { useState } from 'react';
+import { convertToFriendlyWebsiteName } from 'calypso/signup/steps/import/util';
+import ConfirmModal from './confirm-modal';
+
+import './style.scss';
+
+interface Props {
+	fromSite: string;
+	siteSlug: string;
+	startImport: () => void;
+}
+
+export const ImportEverything: React.FunctionComponent< Props > = ( props ) => {
+	const { __ } = useI18n();
+
+	/**
+	 ↓ Fields
+	 */
+	const { fromSite, siteSlug, startImport } = props;
+	const [ isModalDetailsOpen, setIsModalDetailsOpen ] = useState( false );
+
+	return (
+		<>
+			<div className={ classnames( 'import__import-everything' ) }>
+				<div className={ classnames( 'import__site-mapper' ) }>
+					<div className={ classnames( 'import-layout', 'import__site-mapper-border' ) }>
+						<div className={ classnames( 'import-layout__column' ) }>
+							<div
+								className={ classnames(
+									'import-layout__column',
+									'import__site-mapper-header',
+									'import__site-mapper-border'
+								) }
+							>
+								{ __( 'Original site' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Original Site' )` *ES Score: 10*